src/classes/ConfData.js

```diff
@@ -119,7 +119,7 @@ class ConfData {
 			_initProperty('enable_click2play_social', true);
 			_initProperty('enable_human_web', !IS_CLIQZ && !IS_FIREFOX);
 			_initProperty('enable_abtests', true);
-			_initProperty('enable_smart_block', true);
+			_initProperty('enable_smart_browse', true);
```

Comment:
This is going to override the setting for any users with Smart Block currently turned off.  Is there a reason we are changing all of these variable and file names, instead of just changing the user-facing language?